unitTests/ui/AnalyticsTest.ts

```diff
@@ -65,6 +65,10 @@ export function AnalyticsTest() {
         test = null;
       });
 
+      it('use URL from default endpoint if not specified', () => {
+        expect(test.cmp.options.endpoint).toBe('some/uri/rest/ua');
```

Comment:
Would it be possible to reference the `some/uri`  value rather than hard-coding it?